__tests__/app.test.js

```diff
@@ -26,4 +26,41 @@ describe("get /api/categories", () => {
                 expect(res.body.categories.length).toBe(4);
             })
     })
+})
+
+describe("get /api/reviews/review_id", () => {
+    test("responds with JSON containing the expected review according to the review ID", () => {
+        return request(app)
+            .get("/api/reviews/1")
+            .expect(200)
+            .then((response) => {
+                expect(response.body.review).toMatchObject({
+                    review_id: 1,

```

Comment:
If we are trying to get a single review, we would hope the data would be exactly that data, so here we should check the exact title, etc.